yellowbrick/features/pca.py

```diff
@@ -154,7 +155,7 @@ def transform(self, X, y=None, **kwargs):
     def draw(self, **kwargs):
         X = self.pca_features_
         if self.proj_dim == 2:
-            self.ax.scatter(X[:, 0], X[:, 1], c=self.color, cmap=self.colormap)
+            self.ax.scatter(X[:, 0], X[:, 1], c=self.color, cmap=self.colormap,alpha=self.alpha)
```

Comment:
place a space after the comma right before alpha

Reply:
OK. Will do so!